modules/gapi/include/opencv2/gapi/infer/ie.hpp

```diff
@@ -67,6 +67,19 @@ namespace detail {
         Kind kind;
         bool is_generic;
         IEConfig config;
+
+        ParamDesc(const std::string& model, const std::string& weights, const std::string& device,
```

Comment:
Why is it needed ?

Reply:
Understood, but you make copy here

Reply:
fixed, pls check if OK

Reply:
returned to const references https://github.com/opencv/opencv/pull/18913#discussion_r542640719